deploy/manifests/nginx-gateway.yaml

```diff
@@ -100,7 +100,7 @@ subjects:
   name: nginx-gateway
```

Comment:
Curious about all the `nginx-gateway` names. Do these need to be changed too?

Reply:
I think we can leave that as-is.